app/assets/stylesheets/reminders.scss

```diff
@@ -0,0 +1,8 @@
+.reminder_info {
+  padding-top: 10px;
+}
+
+span .reminder_info_in_effect {
+  padding-top: 10px;
+  background-color: #C9D6AE;
```

Comment:
Color literals like `#C9D6AE` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#C9D6AE` should be written as `#c9d6ae`